### PR TITLE
Individual Documentation update for v1.3b

### LINKED ISSUE #103: Implement sort-strength and sort-weakness into User Guide and Developer Guide
1. Function definitions
2. Use cases

### PR DESCRIPTION
**Pull request by AAlghrairy**
1. Related to #71 
2. Closes #103 and #111 
3. Adds feature definitions for `sort-strength` and `sort-weakness` commands in User Guide
4. Include rational for Strength and Weakness functions in Q&A section of User Guide
5. Adds use cases and a user story to Developer Guide
6. Adds additional fields to individual PPP

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -467,22 +467,22 @@ schedules, and provides them with a platform to visualise defensive and offensiv
...
+| `* * *`  | disorganized                    | add and tag new roles/teams to a contact                        | easily retrieve relevant information                                         |
+| `* * *`  | coach                           | easily retrieve contact information of relevant parties         | quickly broadcast information to them                                        |
+| `* * *`  | organized coach                 | view players by their strengths and weaknesses                  | make informed decision on choosing the best person                           |
+| `* * *`  | disorganized coach              | add existing and upcoming tasks                                 | keep track of my schedule easily                                             |
+| `* * *`  | disorganized coach              | delete wrongly added or past tasks                              | organize my schedule better                                                  |
+| `* * *`  | strategic coach                 | save current formation information                              | plan my game plays and save strategies                                       |
```
Comment (simjunheng) [inline docs/DeveloperGuide.md:480]: I think this user story can be removed as we are not planning to implement the save formation feature

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -467,22 +467,22 @@ schedules, and provides them with a platform to visualise defensive and offensiv
...
+| `* *`    | lazy and forgetful coach        | view a list of help commands and their descriptions             | easily recall how to do a specific task                                      |
+| `* *`    | organised coach                 | view players by their strengths and weaknesses                  | make informed decisions on choosing the best person for a specific objective |
+| `**`     | organized and data-driven coach | sort my players by strengths and weaknesses                     | get a better understanding of how players compare                            |
+| `*`      | strategic coach                 | change the position of players (x-y coordinate) during the game | ensure my team works together                                                |
+| `*`      | coach                           | drag and drop a player into a calendar                          | plan scheduled events for them according to their needs                      |
```
Comment (simjunheng) [inline docs/DeveloperGuide.md:485]: I think this user story can be removed as we are not planning to implement a calendar in our program

File: docs/UserGuide.md [hunk 3]
```diff
@@ -466,31 +478,34 @@ Format: `exit`
...
+| **Edit**               | `edit-p INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [t/TAG_NAME]…​` <br> e.g., `edit-p 1 p/98273712 e/johndoe@example.com`                                   |
+| **Clear**              | `clear-p`                                                                                                                                                     |
+| **Add Tag**            | `tag-add-p INDEX TAG_NAME`<br> e.g., `tag-add-p 1 Public Relations`                                                                                           |
+| **Del Tag**            | `tag-del-p INDEX TAG_NAME`<br> e.g., `tag-del-p 1 Team`                                                                                                       |
+| **Find**               | `find-p [n/NAME]…​ [t/TAG]…​`<br> e.g., `find-p n/Alex n/Charlotte t/team1 t/team2`                                                                     |
+| **Add Note**           | `note-add INDEX  NOTE_DESCRIPTION`<br> e.g., `note-add 1 Surgery Scheduled for tomorrow`                                                                      |
```
Comment (simjunheng) [inline docs/UserGuide.md:499]: Just to check, there is no longer a `note-add` and `note-del` command right?

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -657,6 +657,28 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 1d1. Coach2K22 shows an error message.
 
       Use case ends.    
+
+**Use case: Sort address book by strengths in descending order**
+
+**MSS**
+
+1. User requests to sort the list of persons by total strengths
```
Comment (simjunheng) [inline docs/DeveloperGuide.md:665]: Would `total number of strengths` sound better?

**Review by simjunheng — APPROVED** [review]
LGTM except for some minor details

File: docs/UserGuide.md (hunk 3 above)
Comment (AAlghrairy) [reply docs/UserGuide.md:499]: I'll remove that now. Thanks for catching it 👍

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (AAlghrairy) [reply docs/DeveloperGuide.md:485]: Is it necessary to delete given that it's given low priority (under the section "unlikely to have"?

Comment (simjunheng) [reply docs/DeveloperGuide.md:484]: You're right. I guess we can keep this user story

**Pull request merged**